### PR TITLE
[AR-1853] Update Datadog integration docs for new metric names

### PR DESCRIPTION
**Pull request by glasser**

### CODE REVIEW

**Comment by glasser** [comment]
Note: the first image needs to be updated to show the new integration name once Datadog updates their site.

File: studio-docs/source/datadog-integration.md [hunk 1]
```diff
@@ -42,29 +42,31 @@ Studio forwards the following metrics to Datadog:
...
+|`apollo.operations.latency.min`<br/>`apollo.operations.latency.median`<br/>`apollo.operations.latency.95percentile`<br/>`apollo.operations.latency.99percentile`<br/>`apollo.operations.latency.max`<br/>`apollo.operations.latency.avg`| A histogram of GraphQL operation response times, measured in milliseconds. Because of Studio's aggregation method (logarithmic binning), these values are accurate to +/- 5%. |
 
 
 These metrics are aggregated in 60-second intervals and tagged with the GraphQL operation name as `operation:<query-name>`. Unique query signatures with the same operation name are merged, and queries without an operation name are ignored.
 
-These metrics are also tagged with both the associated Studio graph ID (as `service:<graph-id>`) and the associated variant name (as `variant:<variant-name>`). If you haven't set a variant name, then `current` is used.
+These metrics are also tagged with both the associated Studio graph ID (as `graph:<graph-id>`) and the associated variant name (as `variant:<variant-name>`). If you haven't set a variant name, then `current` is used.
```
Comment (zionts) [inline studio-docs/source/datadog-integration.md:53]: Can we call it an `Apollo graph ID`?

File: studio-docs/source/datadog-integration.md [hunk 2]
```diff
@@ -42,29 +42,31 @@ Studio forwards the following metrics to Datadog:
...
+
+If you set up your integration before October 2020, the metric names start with `apollo.engine.operations` instead of `apollo.operations`, and used a `service:<graph-id>` tag instead of `graph:<graph-id>`. This is called "legacy mode". You may transition your graph to modern mode by clicking the "Transition to modern mode" button on your graph's Integrations page. This is a one-way change; you should be prepared to update any dashboards and metrics to use the new metric and tag names when you click the button.
```
Comment (zionts) [inline studio-docs/source/datadog-integration.md:55]: What do you think about block quoting this?

**Review by zionts — APPROVED** [review]
Excited for this to land!

File: studio-docs/source/datadog-integration.md (hunk 1 above)
Comment (glasser) [reply studio-docs/source/datadog-integration.md:53]: done

File: studio-docs/source/datadog-integration.md (hunk 2 above)
Comment (glasser) [reply studio-docs/source/datadog-integration.md:55]: done

File: studio-docs/source/datadog-integration.md [hunk 3]
```diff
@@ -42,29 +42,31 @@ Studio forwards the following metrics to Datadog:
...
 
 ## Exploring metrics
 
-In the [Datadog metrics explorer](http://app.datadoghq.com/metric/explorer?exp_metric=apollo.engine.operations.count&exp_group=service&exp_agg=sum&exp_row_type=metric), all Studio metrics are [tagged](https://www.datadoghq.com/blog/the-power-of-tagged-metrics/) with the graph ID (`service:<graph-id>`), the variant name (`variant:<variant-name>`), and the operation name (`operation:<query-name>`). These values are normalized according to Datadog naming requirements (all letters are lowercase, and illegal symbols are converted to underscores).
+In the [Datadog metrics explorer](http://app.datadoghq.com/metric/explorer?exp_metric=apollo.operations.count&exp_group=graph&exp_agg=sum&exp_row_type=metric), all Studio metrics are [tagged](https://www.datadoghq.com/blog/the-power-of-tagged-metrics/) with the graph ID (`graph:<graph-id>`), the variant name (`variant:<variant-name>`), and the operation name (`operation:<query-name>`). These values are normalized according to Datadog naming requirements (all letters are lowercase, and illegal symbols are converted to underscores).
```
[the remark this answers is not in the document]
Comment (glasser) [reply studio-docs/source/datadog-integration.md:59]: We do earlier in the doc. I guess there's a bit of duplication between the sections. I think I'll leave it as it is for now though maybe a deeper cleanup would reduce redundancy.

**Comment by glasser** [comment]
> Note: the first image needs to be updated to show the new integration name once Datadog updates their site.

Datadog is seriously lagging on getting the logo and title updated. I'll deal with that later. See https://github.com/DataDog/integrations-extras/pull/729 for details

**Pull request merged**